tests/python/relay/model_zoo_data/lstm_i128_h128-symbol.json.data

```diff
@@ -0,0 +1,290 @@
+{
+  "nodes": [
```

Comment:
binary file should not be part of the repo, generate them on the fly or put them elsewhere

Reply:
I have some clean up commits coming, working on them right now.